libbpf-rs/src/query.rs

```diff
@@ -498,29 +498,80 @@ gen_info_impl!(
 // TODO: Document members.
 #[allow(missing_docs)]
 pub struct BtfInfo {
-    pub btf: u64,
-    pub btf_size: u32,
+    pub name: String,
+    pub btf: Vec<u8>,
```

Comment:
Would you mind terribly documenting these changed members? We try to document everything that is new at the very least in the hopes that eventually we can remove all `#[allow(missing_docs)]`.